src/python/lib/ensembl/compara/utils/taxonomy/taxon_selector.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Collection of taxonomy methods to configure and extract taxonomy related information.
+
+Typical usage examples::
+
+    from ensembl.compara.utils.taxonomy import *
```

Comment:
Do you have another example without `import *`? Like importing a specific function?